internal/cmd/cmd.go

```diff
@@ -306,9 +306,11 @@ func NewRootCmd() *cobra.Command {
 			}
 			if rootOptions.Info {
 				if err := mustBeLoggedIn(); err != nil {
-					return err
+					return fmt.Errorf("login check: %w", err)
+				}
+				if err := info(); err != nil {
+					return fmt.Errorf("info: %w", err)
 				}
-				return info()
```

Comment:
Missing `return nil` here? I don't think we want to print help when `--info` is used, do we?